TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -302,24 +302,24 @@ TYPO3:
       defaultEditPreviewMode: 'inPlace'
       editPreviewModes:
         # Live mode is only configured here for consistency. You shouldn't change it.
-        live:
+        'TYPO3.Neos:live':
```

Comment:
Right, can you point me to an existing configuration migration, not sure how to proceed in clean way.